spec/site_helper.rb

```diff
@@ -0,0 +1,31 @@
+require "yaml"
+require "tempfile"
+
+module Helpers
+module_function
+
+  # Build the example site using middleman
+
+  def rebuild_site!(config: "config/tech-docs.yml", overrides: {})
+    config_file = Tempfile.new("tech_docs_config")
+
+    begin
+      Dir.chdir("example") do
+        puts Dir.getwd
```

Comment:
Is this still needed? Or was this just for debugging?

Reply:
Oh nice spot, thanks!